packages/server/src/sdk/app/applications/sync.ts

```diff
@@ -107,9 +100,11 @@ export async function syncUsersToAllApps(userIds: string[]) {
   }
   const resp = await Promise.allSettled(promises)
   const failed = resp.filter(promise => promise.status === "rejected")
-  if (failed.length > 0) {
-    const reasons = failed.map(fail => (fail as PromiseRejectedResult).reason)
-    logging.logAlert("Failed to sync users to apps", reasons)
+  const reasons = failed
+    .map(fail => (fail as PromiseRejectedResult).reason)
+    .filter(reason => !dbCore.is409Error(reason))
+  if (reasons.length > 0) {
+    logging.logWarn("Failed to sync users to apps", reasons)
```

Comment:
Thought we might `logAlert` here as this will be a non-409 failure

Reply:
Discussed this above, I do not believe it is an error scenario if there is a failure here, I wouldn't want people rushing to look at an error here as it is not critical/important, there are multiple fallbacks and the process will be re-ran on every update.

Reply:
Going to resolve this discussion as further discussion occurring in below thread.